itou/templates/apply/includes/job_applications_filters/top_filters.html

```diff
@@ -1,6 +1,6 @@
 {% load django_bootstrap5 %}
 
-<div class="s-section__row row">
+<div class="s-section__row row hide-on-selection">
```

Comment:
Pourquoi une class css pour gérer un état ? Perso je préfèrerais un `data-it-attr` genre `data-it-hide-on-selection="true"`.  
Bootstrap utilise beaucoup les `data-bs-attr` comme paramètres de configuration js ou autre. Et du coup, j'en utilise pas mal niveau thème, pour respecter la même logique. Comme [ici](https://github.com/gip-inclusion/les-emplois/blob/master/itou/templates/apply/process_details_company.html#L20) ou [là](https://github.com/gip-inclusion/les-emplois/blob/master/itou/templates/includes/copy_to_clipboard.html#L3), par exemple